internal.h

```diff
@@ -27,7 +27,7 @@ typedef struct polygon {
 	int vert[MAXVERTICES];		/* vertex indicies, in clockwise order */
 	double u[MAXVERTICES];		/* texture coordinates (in same order as vertices) */
 	double v[MAXVERTICES];
-} Polygon;
+} _Polygon;
```

Comment:
Could you add a comment here explaining why this is necessary?  I'm assuming it conflicts with something in some windows header, but perhaps it's possible to not include that header instead?

Reply:
You are correct, it is due to conflicts with headers (Visual Studio and/or Windows). I have tried to isolate the problem but was out of luck. I didn't wanted to spend too much time on it because I just wanted to have the tool up and running.
I thought the renaming was not a big issue, but I understand your point of view. I can try to look at the issue in a deeper way.